webrtc.html

```diff
@@ -11641,6 +11643,27 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
                 <td>The data channel has failed.</td>
               </tr>
               <tr>
+                <td><dfn><code>dtls-failure</code></dfn></td>
+                <td>The DTLS negotiation has failed or the connection
+                has been terminated with a fatal error. The
+                <code>message</code> contains information relating to
+                the nature of error.  If a DTLS alert was received,
```

Comment:
May want to clarify that this only happens for fatal alerts: "If a fatal DTLS alert was received... If a fatal DTLS alert was sent..."